assembly/std/address.ts

```diff
@@ -1,10 +1,10 @@
-import { Valider, ByteArray } from '@massalabs/as-types';
+import { Valider, Serializable, Args, Result } from '@massalabs/as-types';
 
 /**
  * A Massa's blockchain address.
  *
  */
-export class Address implements Valider {
+export class Address implements Valider, Serializable {
```

Comment:
Oh shit Valider still exist ? Maybe we should mark it a deprecated.

Reply:
okay, issue create here: https://github.com/massalabs/as/issues/92